sdk/notificationhubs/notification-hubs/samples-dev/createInstallation.fcmLegacy.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how the createOrUpdateInstallation() method can be used to create or overwrite an
+ * installation in place.
+ *
+ * See https://docs.microsoft.com/azure/notification-hubs/notification-hubs-push-notification-registration-management
+ * to learn about installations.
+ *
+ *
+ * @summary Demonstrates how to create or overwrite an installation using Azure Notification Hubs
+ * @azsdk-weight 100
+ */
+
+import { createClientContext } from "@azure/notification-hubs/client";
+import { createFcmLegacyInstallation } from "@azure/notification-hubs/models/installation";
+import { createOrUpdateInstallation } from "@azure/notification-hubs/client/createOrUpdateInstallation";
```

Comment:
should the samples have a single import line assuming these path imports are more of an advanced scenario?

Reply:
These samples use the modular approach.  I suppose I should add samples using the classical approach as well.  Will do that before the next Beta